acceptance_tests/features/environment.py

```diff
@@ -40,6 +41,17 @@ def before_scenario(_, scenario):
         return
 
 
+def before_step(context, step):
+    try:
+        context.survey, context.survey_period = next(tag.split('_')[1:] for tag in context.tags if tag.startswith('ce'))
```

Comment:
Would the correct way be to parameterise the collection exercise? https://jenisys.github.io/behave.example/tutorials/tutorial03.html

Reply:
That would be much more preferable. I just wasn't sure about inserting additional (context) data into the defined acceptance criteria.